examples/Services/AccountManagement/GetAccessibleCustomers.cs

```diff
@@ -0,0 +1,92 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Ads.GoogleAds.Lib;
+using Google.Ads.GoogleAds.V0.Resources;
+using Google.Ads.GoogleAds.V0.Services;
+
+using System;
+
+namespace Google.Ads.GoogleAds.Examples.V0
+{
+    /// <summary>
+    /// This code example lists the resource names for the customers that the authenticating user
+    /// has access to. This example may can be used to display a list of customer IDs that can be
+    /// selected by the user for setting the login-customer-id in a subsequent call. See
+    /// https://developers.google.com/google-ads/api/docs/concepts/call-structure#login-customer-id
+    /// to learn more about login-customer-id header.
+    /// </summary>
+    public class GetAccessibleCustomers : ExampleBase
+    {
+        /// <summary>
+        /// Main method, to run this code example as a standalone application.
+        /// </summary>
+        /// <param name="args">The command line arguments.</param>
+        public static void Main(string[] args)
+        {
+            GetAccessibleCustomers codeExample = new GetAccessibleCustomers();
+
+            Console.WriteLine(codeExample.Description);
+            codeExample.Run(new GoogleAdsClient());
+        }
+
+        /// <summary>
+        /// Returns a description about the code example.
+        /// </summary>
+        public override string Description
+        {
+            get
+            {
+                return "This code example lists the resource names for the customers that the " +
+                    "authenticating user has access to. This example may be used to display a " +
+                    "list of customer IDs that can be selected by the user for setting the " +
+                    "login-customer-id in a subsequent call. See "+
+                    "https://developers.google.com/google-ads/api/docs/concepts/call-structure#login-customer-id "+
+                    "to learn more about login-customer-id header.";
+            }
+        }
+
+        /// <summary>
+        /// Runs the code example.
+        /// </summary>
+        /// <param name="client">The Google Ads client.</param>
+        public void Run(GoogleAdsClient client)
+        {
+            // Get the CustomerService.
+            CustomerServiceClient customerService = client.GetService(Services.V0.CustomerService);
+
+            try
+            {
+                // Retrieve the list of customer resources.
+                string[] customerResourceNames = customerService.ListAccessibleCustomers();
+
+                Console.WriteLine($"Total results: {customerResourceNames.Length}.");
+
+                // Display the result.
+                int i = 1;
+                foreach (string customerResourceName in customerResourceNames)
+                {
+                    Console.WriteLine($"{i++}) Customer resource name: {customerResourceName}");
```

Comment:
Let's be consistent about how we print lists in examples (I mentioned this on the other PR as well).

Reply:
Done.

Reply:
Done